lib/registries/addon/components/sharing-icons/popover/styles.scss

```diff
@@ -0,0 +1,50 @@
+
+:global(.dropdown-menu).SharingIconsPopover {
+    background-color: #fff;
+    border: 1px solid #ccc;
+    border-radius: 2px;
+    min-width: initial;
+    min-height: initial;
+    width: 105px;
+    height: 36px;
+    margin-right: 30px;
+    top: -7px;
+    left: -105px;
+    text-align: center;
+}
+
+:global(.btn-link).SharingIconsPopover__button {
+    background-color: rgba(0, 0, 0, 0) !important;
```

Comment:
I hate every one of these importants. If able to switch to something other than dropdown, these might be killable